addons/point_of_sale/static/src/app/generic_components/orderline/orderline.xml

```diff
@@ -39,10 +39,8 @@
                         <i class="fa fa-sticky-note me-1" role="img" aria-label="Customer Note" title="Customer Note"/>
                         <t t-esc="line.customerNote" />
                     </li>
-                    <div class="internal-note-container d-flex gap-2">
-                        <t t-foreach="line.internalNote?.split?.('\n') or []" t-as="note" t-key="note_index">
-                            <li t-if="note.trim() !== ''" t-esc="note" class="internal-note badge mt-2 p-2 rounded-pill bg-info text-info bg-opacity-25" style="font-size: 0.85rem;" />
-                        </t>
+                    <div t-if="line.internalNote" class="d-flex flex-wrap gap-1 pt-2 bg-opacity-75">
```

Comment:
```suggestion
                    <div t-if="line.internalNote" class="pt-2 bg-opacity-75">
```
do you need the `d-flex` here ?

Reply:
Yes We can remove that but i don't find any way to give gap between two notes
![image](https://github.com/user-attachments/assets/cca44f3f-139d-4e1b-955c-793e8223890e)
as the notes css are directly coming from Tagslist which will be spans containing this name of note with css but no margin between them